extensions/libarchive/BinFiles.h

```diff
@@ -91,7 +92,7 @@ class Bin {
       if (flow->getAttribute(fileCount_, value)) {
         try {
           // for defrag case using the identification
-          int count = std::stoi(value);
+          size_t count = std::stoul(value);
```

Comment:
could use 
```
uint32_t count;
utils::internal::ValueParser(value).parse(count).parseEnd()
```

Reply:
no `size_t` overload unfortunately

Reply:
I wanted to keep changes minimal, just making sure warnings, especially warnings about implicit narrowing, are gone.